activities/trainings/writing-issues.md

```diff
@@ -0,0 +1,73 @@
+---
+type: Guide
+explains: How to clearly communicate what is wrong and suggest fixes
+---
+
+# Writing issues
+
+An issue is a story explaining why something should be changed. It serves as the seed for collaboration in order to get it solved.
+
+Issues can be of two types:
+
+* Bug: something that isn't as it is supposed to be, this is broken
+* Enhancement: an improvement that could be done to make it better
+
+The goal of an issue is to provide knowledge about and context on what to fix or improve. You can make issues alone or together.
+
+An issue can contain a proposed fix, but does not have to.
+
+We write good issues because:
+
+* it helps communicate things that need to be done
+* planning becomes easier if you understand the context of a change
+* the person filing an issue is not always the person to fix it, they might have different skillsets
+* commit messages become better with more context
+
+We use the list of issues to:
+
+* understand what needs to be done
+* prioritize what to do
+* collaborating with each other on finding solutions
+* help new contributors find out how they can contribute
+
+## Structure
+
+The title of the issue should be short, to the point an descriptive. Enhancements can have verbs in them. Detail, technical or otherwise, should be in the body of the issue.
```

Comment:
```suggestion
The title of the issue should be short, to the point and descriptive. Enhancements can have verbs in them. Detail, technical or otherwise, should be in the body of the issue.
```